src/api/app/views/webui2/webui/attribute/_form.html.haml

```diff
@@ -0,0 +1,33 @@
+- value_fields_locals = { attribute: attribute, allowed_values: allowed_values }
+- issue_fields_locals = { issue_trackers: issue_trackers }
+
+= form_for(attribute) do |form|
+  = form.hidden_field(:attrib_type_id)
+
+  - errors = attribute.errors
```

Comment:
[nitpick] I don't see the need for this variable...

Reply:
I use it in 3 different spots, I prefer it this way.